### PR TITLE
Adds cat tongue

### PR DESCRIPTION
**Pull request by yorii**
## About The Pull Request

Closes #3930 

Correctly places the say_mod of felinids under their tongue, as would be expected.

## Why It's Good For The Game

This just makes more sense, and now you have another reason to surgically replace tongues.

## Changelog
:cl:
add: added cat tongues to felinids
/:cl:

### CODE REVIEW

**Comment by eeSPee** [comment]
Make cat surgeon replace your tongue too.

**Comment by yorii** [comment]
> Make cat surgeon replace your tongue too.

Excellent idea!

**Comment by ItsTisme** [comment]
Now to port furranium from citadel to give you furry tongues

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -14,6 +13,8 @@
 
 	mutantears = /obj/item/organ/ears/cat
 	mutanttail = /obj/item/organ/tail/cat
+	mutanttongue = /obj/item/organ/tongue/cat
```
Comment (Archanial) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:16]: Who approved that?

File: code/modules/mob/living/simple_animal/hostile/cat_butcher.dm [hunk 2]
```diff
@@ -80,7 +80,18 @@
...
 				return
-		else if(!L.has_trauma_type(/datum/brain_trauma/severe/pacifism) && L.getorgan(/obj/item/organ/ears/cat) && L.getorgan(/obj/item/organ/tail/cat)) //still does damage. This also lacks a Stat check- felinids beware.
+		else if(!L.getorgan(/obj/item/organ/tongue/cat) && L.stat)
+			if(L.getorgan(/obj/item/organ/tongue))
+				var/obj/item/organ/tongue/tongue = L.getorgan(/obj/item/organ/tongue)
+				visible_message("[src] slices off [L]'s tongue!", "<span class='notice'>You slice [L]'s tongue off.</span>")
+				tongue.Remove(L)
+				tongue.forceMove(get_turf(L))
+			else
+				visible_message("[src] implants a cat tongue into [L]!", "<span class='notice'>You implant a cat tongue into [L].</span>")
```
Comment (ItsTisme) [inline code/modules/mob/living/simple_animal/hostile/cat_butcher.dm:90]: maby change "cat tongue" to the organ name to be more generic incase we want more tongue types in the future

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 1 above)
Comment (yorii) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:16]: I have approval for a non-specific felinid change PR from TheFakeElon:
image

File: code/modules/mob/living/simple_animal/hostile/cat_butcher.dm (hunk 2 above)
Comment (yorii) [reply code/modules/mob/living/simple_animal/hostile/cat_butcher.dm:90]: yeah but, this message only ever comes from the cat butcher, and all he ever does is convert you to a cat

**Comment by PowerfulBacon** [comment]
Its funny because their species gimmick is that they are annoying.

**Comment by Sinestia** [comment]
I have to ask, why?


 This makes it even more inconsistent too unless you apply it to other species, which at least would help prepare things for zams' proposed species rework

**Comment by yorii** [comment]
> This makes it even more inconsistent too unless you apply it to other species, which at least would help prepare things for zams' proposed species rework

It was already applied decently consistently to my great surprise, but yes I do intend to give all organ code a workover unless zams does so first.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Remove the speech handling or make it not default.

**Comment by Sinestia** [comment]
Bacon desperately trying to get rid of the say mod yet again tm

**Comment by PowerfulBacon** [comment]
> Bacon desperately trying to get rid of the say mod yet again tm

lol that has nothing to do with the say mod. (Also without the say mod it would still be tied to the species not the tongue so it wouldn't matter unless overridden by the tongue.)

**Comment by Sinestia** [comment]
My bad, misread you then and didnt notice handle_speech 😔

**Comment by yorii** [comment]
> Remove the speech handling or make it not default.

make it not default..? how do you mean, make it triggered by some other factor or configurable with a config flag?

**Comment by PowerfulBacon** [comment]
I dunno making it part of cat surgeon or dropped by cats and require surgery.

**Comment by PowerfulBacon** [comment]
Race changes need approval and stuff.

**Comment by PowerfulBacon** [comment]
That on top of the 1% uwu talk is good enough reason to close.

**Comment by yorii** [comment]
@PowerfulBacon but I did get approval..

**Pull request closed without merging**